extension/src/lib.rs

```diff
@@ -8,8 +11,25 @@ mod type_builder;
 mod serialization;
 mod schema_test;
 
+use pgx::*;
+
 pgx::pg_module_magic!();
 
+static EXPERIMENTAL_ENABLED: GucSetting<bool> = GucSetting::new(false);
+
+#[pg_guard]
+pub extern "C" fn _PG_init() {
+    GucRegistry::define_bool_guc(
+        "timescale_analytics_acknowledge_auto_drop",
+        "enable creation of auto-dropping objects using experimental timescale_analytics_features",
+        "experimental features are very unstable, and objects \
+            depending on them will be automatically deleted on extension update",
+        &EXPERIMENTAL_ENABLED,
+        //TODO should this be superuser?
```

Comment:
No. Won't work on Forge I think.

Reply:
also now that it's not a different extension, I think it's fine